apps/financial-aid/web-osk/src/components/DirectTaxPaymentsModal/DirectTaxPaymentModal.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import {
+  ModalBase,
+  Text,
+  Box,
+  Button,
+  Divider,
+} from '@island.is/island-ui/core'
+
+import * as styles from './DirectTaxPaymentModal.css'
+import {
+  TaxBreakdownItem,
+  TaxBreakdownHeaders,
+  TaxBreakdown,
+} from '@island.is/financial-aid/shared/components'
+import {
+  DirectTaxPayment,
+  formatNationalId,
+  getMonth,
+} from '@island.is/financial-aid/shared/lib'
+
+interface Props {
+  isVisible: boolean
+  onVisibilityChange: React.Dispatch<React.SetStateAction<boolean>>
+  items: DirectTaxPayment[]
+}
+
+const DirectTaxPaymentModal = ({
```

Comment:
How is this component looking in mobile? Too crowded or?

Reply:
It is a bit crowded, pair on it together? 🙏